packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1966,6 +1950,41 @@ class GridPlacementData<DataType> extends ParentDataWidget<GridRenderObjectWidge
 ///
 /// If you only have one child, then rather than using [Flex], [Row], or
 /// [Column], consider using [Align] or [Center] to position the child.
+///
+/// ## Layout algorithm
+///
+/// Layout for a [Flex] proceeds in six steps:
+///
+/// 1. Layout each child a null or zero flex factor (i.e., those that are not
+///    [Expanded]) with unbounded main axis constraints and the incoming
```

Comment:
since you've kept Flexible and not marked it deprecated, this is confusing. It's not only Expanded, it's also Flexible.